### PR TITLE
Move eips and bandwidths to networking/v1

### PR DESCRIPTION
**Pull request by niuzhenguo**
**What this PR does / why we need it**:
In case of collisions with vpc resources, move eips and bandwidths
to networking from vpc.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

Partially address #4 

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: openstack/client_extension.go [hunk 1]
```diff
@@ -113,10 +113,11 @@ func NewElasticLoadBalancer(client *golangsdk.ProviderClient, eo golangsdk.Endpo
 	return sc, err
 }
 
-//NewVpcServiceV1 creates the a ServiceClient that may be used to access the v1
-//vpc service which is a service of public ip management of huawei cloud
-func NewVpcServiceV1(client *golangsdk.ProviderClient, eo golangsdk.EndpointOpts) (*golangsdk.ServiceClient, error) {
+// NewNetworkV1 creates a ServiceClient that may be used with the v1 network
+// package.
+func NewNetworkV1(client *golangsdk.ProviderClient, eo golangsdk.EndpointOpts) (*golangsdk.ServiceClient, error) {
```
Comment (freesky-edward) [inline openstack/client_extension.go]: we'd better bring this into correspondence with other VPC resource e.g. https://github.com/gator1/gophercloud/blob/master/openstack/client.go#L330:6

Comment (niuzhenguo) [reply openstack/client_extension.go]: sure, as we agreed to place all such resources under networking/v1 and the endpoint type is named 'network', so seems the client name here is better?

Comment (freesky-edward) [reply openstack/client_extension.go]: client name looks good to me, there are some difference of ResourceBase. how to deal with project id here?

Comment (niuzhenguo) [reply openstack/client_extension.go]: If not all resources require a project id with the endpoint, we should just let themselves deal with it in their urls.go like  https://github.com/huaweicloud/golangsdk/blob/master/openstack/vpc/v1/eips/urls.go#L9

Comment (freesky-edward) [reply openstack/client_extension.go]: agree

**Pull request merged**